.github/workflows/client-build.yml

```diff
@@ -0,0 +1,58 @@
+name: Client
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+env:
+  cwd: ${{github.workspace}}/packages/client
+
+defaults:
+  run:
+    working-directory: packages/client
+
+jobs:
+  test-client:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      
+      - name: Dependency cache
+        uses: actions/cache@v2
+        id: cache
+        with:
+          key: Client-${{ runner.os }}-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+          path: '**/node_modules'
+
+      # Installs root dependencies, ignoring Bootstrap All script.
+      # Bootstraps the current package only
+      - run: npm install --ignore-scripts && npm run bootstrap:client
+        if: steps.cache.outputs.cache-hit != 'true'
+        working-directory: ${{github.workspace}}
+
+      # Builds current package and the ones it depends from.
+      - run: npm run build:client
```

Comment:
for the client specifically I don't believe we need to build before running `npm run coverage`, that is needed for packages that depend on others.

Reply:
This is done in all other build job files like `block-build.yml`, wouldn't want to remove here just for the client. If you think this is misplaced or not necessary maybe rather tackle along a separate PR.